kibbeh/src/ui/MinimizedRoomCard.tsx

```diff
@@ -29,19 +29,19 @@ export const MinimizedRoomCard: React.FC<MinimizedRoomCardProps> = ({
   room,
 }) => {
   const router = useRouter();
-
+  // gap-n only works with grid
```

Comment:
No it doesn't? We have flexbox by default, and `gap` is universal between flex and grid

Reply:
https://tailwindcss.com/docs/gap

Reply:
https://css-tricks.com/almanac/properties/g/gap/

Reply:
![Overlisted looking at TailWind: Is this CSS?](https://user-images.githubusercontent.com/54026315/112604246-3421ff00-8e16-11eb-92a9-710163cd18df.png)

Reply:
Wait a minute....

Chromium: 
![image](https://user-images.githubusercontent.com/54026315/112607299-59fcd300-8e19-11eb-9ccb-47d25b5da7e3.png)

Safari: 
![Snímek obrazovky 2021-03-26 v 9 43 06](https://user-images.githubusercontent.com/54026315/112607324-5ff2b400-8e19-11eb-8d9e-def7f40a7249.png)

I am sorry Overlisted